examples/mobile/HomeApp/index.html

```diff
@@ -34,8 +34,10 @@
 		</header>
 		<div class="ui-content">
 			<div class="ui-content-area">
-				Weather Widget
-			</div>
+				<a class="weatherwidget-io" href="https://forecast7.com/en/37d57126d98/seoul/" data-label_1="Seoul" data-label_2="South Korea" data-theme="original" >Seoul South Korea</a>
```

Comment:
It's just enough image ( It don't make real widget, not need url link ^^).
We need to more focus on the sophisticated  GUI look&feel, not functionality.
so we need to make the round corner of it.

Reply:
advantage of this widget is that it is responsive. I can place image but I don't know how this will look on the demo device.
In case of image content will be probably extended or squeezed. I afraid that this will not look nice.

Reply:
ok. Let's use this widgets.
The weather widget image is good.
Just in case, I also try go get the extensible image from UX team.